### DISCUSSION TITLE
cant' enable request API

### DISCUSSION THREAD

**Discussion by peterroe** [tauri-apps/tauri#3779] [Q&A]
I want to use `fetch` to access all URLs without restriction, but it still reports an error:

```shell
url not allowed on the configured scope: https://github.com/peterroe
````

This is my tauri.config.json:

```json
"allowlist": {
  "all": true,
  "http": {
    "all": true,
    "request": true 
  }
}
```

**Comment by errmayank** [comment]
You also need to add `scope` in your `tauri.config.json` which accepts glob pattern for url allowlist, something like this would work:

```json
"allowlist": {  
  "all": true,
  "http": {
    "all": true,
    "request": true,
    "scope": ["https://github.com/**"]
  }
},
```

**Reply by peterroe** [reply]
Thanks, I have solved this problem😊
